runbotics/runbotics-scheduler/src/scheduler-database/process-instance/process-instance.controller.ts

```diff
@@ -36,21 +36,24 @@ export class ProcessInstanceController {
     getPage(
         @Specifiable(ProcessInstanceCriteria) specs: Specs<ProcessInstance>,
         @Pageable() paging: Paging,
-        @UserDecorator() user: User
+        @UserDecorator() user: User,
     ) {
         return this.processInstanceService.getPage(user, specs, paging);
     }
 
     @Get(':id')
-    getOne(@Param('id') id: ProcessInstance['id'], @UserDecorator() user: User) {
+    getOne(
+        @Param('id') id: ProcessInstance['id'],
+        @UserDecorator() user: User,
+    ) {
         return this.processInstanceService.getOne(id, user);
     }
 
     @Get(':id/subprocesses/GetPage')
     getSubprocesses(
         @Param('id') id: ProcessInstance['id'],
         @Pageable() paging: Paging,
-        @UserDecorator() user: User
+        @UserDecorator() user: User,
```

Comment:
do you have some extension or is it ESlint?

Reply:
that adds comma

Reply:
indeed, my hand is an extension :D

Reply:
but I think you can configure prettier in the same way

Reply:
woooooow, nice
![image](https://github.com/user-attachments/assets/ada3a3f7-b53a-406a-b678-a23a6b0e9ca6)